scripts/functions/requirements/osx_brew

```diff
@@ -39,7 +39,13 @@ requirements_osx_brew_lib_installed_prefix_check()
 requirements_osx_brew_lib_installed()
 {
   \typeset brew_lib_prefix
-  brew list -1 | __rvm_grep "^${1}$" >/dev/null &&
+  # Test for older versions of Homebrew before adding the `--formula` flag.
+  brew_version=$( brew --version | tee | head -n 1 | sed 's/[^0-9.]*\([0-9.]*\).*/\1/' )
```

Comment:
`tee` was necessary to prevent a broken pipe, since the brew command will return one line before proceeding to check other versions of casks. If there's a preferred way of doing this, happy to adapt.

Reply:
I know the PR is merged, but if you can use AWK it gets cleaner:

```bash
 brew --version | awk 'NR==1 { print $2 }'
 ```

Reply:
Do you wanna fire up PR with this change?

Reply:
Sure thing! I wasn't sure if it was a change worth enough to open a PR, but I'll make one now :)

Reply:
Does not hurt for sure :) Thanks!